### PR TITLE
Replace Instant Uploads feature with Camera Uploads feature

### PR DESCRIPTION
**Pull request by settermjd**
### References

https://github.com/owncloud/documentation/issues/3709.

### CODE REVIEW

File: user_manual/android_app.rst [hunk 1]
```diff
@@ -228,7 +228,7 @@ Uploading Files Taken From the Camera
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Pictures and videos can be uploaded from your smartphone after choosing the folder where they are stored.
-To specify where they are located, in the *"Settings"* options, under `Instant Uploads`_, enable one of *"Instant picture uploads"* or *"Instant video uploads"*. 
+To specify where they are located, in the *"Settings"* options, under `Camera uploads`_, enable one of *"Instant picture uploads"* or *"Instant video uploads"*.
```
Comment (davigonz) [inline user_manual/android_app.rst]: We have to replace also this "Instant picture/video uploads" occurrences

**Comment by davigonz** [comment]
@settermjd There are more things to change: 

-  Two screenshots which include Instant uploads naming, in **Uploading Files Taken From the Camera** and **Application Settings** sections

- Instant uploads occurrence in list after **However, the ownCloud Android app offers some advantages over the Web interface:** sentence

**Comment by davigonz** [comment]
@settermjd what's the status of this? We are finishing QA stage, the next version will be ready soon.

File: user_manual/android_app.rst (hunk 1 above)
Comment (settermjd) [reply user_manual/android_app.rst]: Thanks for catching that!

**Comment by settermjd** [comment]
I'll try and get it sorted today.

**Comment by davigonz** [comment]
> I'll try and get it sorted today.

👍 Thank you

**Comment by settermjd** [comment]
@davigonz, I've just updated the content, based on your feedback. Can you review again, pls?

**Comment by davigonz** [comment]
> I've just updated the content, based on your feedback. Can you review again, pls?

@settermjd The screenshot in Application Settings section also needs replacing with the new one.

**Comment by settermjd** [comment]
@davigonz, I have replaced the next image with the one that you linked to. Do you have a replacement image for the one in the "Application Settings" section?

**Comment by davigonz** [comment]
>  I have replaced the next image with the one that you linked to. Do you have a replacement image for the one in the "Application Settings" section?

@settermjd You can use this one: https://cloud.owncloud.com/index.php/f/4279999

**Pull request merged**